packages/dd-trace/src/plugins/plugin.js

```diff
@@ -70,6 +70,7 @@ module.exports = class Plugin {
 
   // TODO: Implement filters on resource name for all plugins.
   /** Prevents creation of spans here and for all async descendants. */
+
```

Comment:
Unrelated change that could cause conflicts down the line, better to remove this from the PR instead.